crates/miden-testing/src/mock_chain/chain.rs

```diff
@@ -606,15 +612,15 @@ impl MockChain {
     // INPUTS APIS
     // ----------------------------------------------------------------------------------------
 
-    /// Returns a valid [`TransactionInputs`] for the specified entities, executing against a
-    /// specific block number.
-    pub fn get_transaction_inputs_at(
+    /// Returns a valid [`TransactionKernelInputs`] for the specified entities, executing against
+    /// a specific block number.
+    pub fn get_transaction_tx_inputs_at(
```

Comment:
This probably should be just `get_transaction_inputs_at()` (`transaction_tx_inputs` sounds a bit repetitive).